src/attr/_make.py

```diff
@@ -194,6 +203,21 @@ def wrap(cl):
             cl = _add_hash(cl)
         if init is True:
             cl = _add_init(cl)
+        if slots:
+            cl_dict = dict(cl.__dict__)
+            cl_dict['__slots__'] = tuple(ca_list)
```

Comment:
*cough* ' → "

(yes I *do* feel like a dick pointing this out, thank you for asking ;))